packages/slice-machine/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,108 @@
+import { Text } from "@prismicio/editor-ui";
+import { clsx } from "clsx";
+import {
+  type CSSProperties,
+  type FC,
+  type HTMLAttributes,
+  type ImgHTMLAttributes,
+  type MouseEvent,
+  type PropsWithChildren,
+  type ReactNode,
+  createElement,
+} from "react";
+
+import * as styles from "./Card.css";
+
+type CardProps = PropsWithChildren<
+  {
+    size?: keyof typeof styles.size;
+    style?: CSSProperties;
+    variant?: keyof typeof styles.variant;
+  } & (
+    | {
+        interactive: true;
+        checked?: boolean;
+        disabled?: boolean;
+        onClick?: (event: MouseEvent) => void;
+      }
+    | {
+        interactive?: false;
+        checked?: never;
+        disabled?: never;
+        onClick?: never;
+      }
+  )
+>;
+
+export const Card: FC<CardProps> = (props) => {
+  const {
+    checked: _checked,
```

Comment:
⚠️ Why extracting it with "_" to remove it from otherProps, while you would extract it normally and using it instead of doing `props.checked`. No?

Reply:
Ok good one sorry, thanks for the explanation 🙂